tools/bin/mbedtls-test-language-server

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+"""Language server for Mbed TLS test files
+"""
```

Comment:
Can you please share some basic usage instructions? How can users of Vim and VScode make us of this?